src/target/intrin_rule.cc

```diff
@@ -24,179 +24,234 @@
 #include "intrin_rule.h"
 
 #include <tvm/tir/op.h>
+#include <tvm/tir/op_attr_types.h>
 
 namespace tvm {
 namespace codegen {
 namespace intrin {
+using tir::FLowerIntrinsic;
 
-TVM_REGISTER_GLOBAL("tvm.intrin.rule.default.exp").set_body(DispatchPureExtern<FloatSuffix>);
+TVM_REGISTER_OP("tir.exp").set_attr<FLowerIntrinsic>("default.FLowerIntrinsic",
```

Comment:
shall we remove this file to, like, `lower_intrinsic_default.cc`? CC: @tqchen

Reply:
I think the naming is just fine because the `lower_intrinsic.cc`file kind of implies it's default rules.